Dockerfile

```diff
@@ -9,7 +9,10 @@ RUN make -C fortio submodule
 # Demonstrate moving the static directory outside of the go source tree and
 # the default data directory to a /var/lib/... volume
 RUN CGO_ENABLED=0 GOOS=linux go build -a -ldflags \
-  '-X istio.io/fortio/ui.resourcesDir=/usr/local/lib/fortio -X main.defaultDataDir=/var/lib/istio/fortio -s' \
+  "-s -X istio.io/fortio/ui.resourcesDir=/usr/local/lib/fortio -X main.defaultDataDir=/var/lib/istio/fortio \
+  -X istio.io/fortio/version.buildInfo=$(date +%y%m%d_%H%M_)$(cd fortio; git rev-parse HEAD) \
```

Comment:
https://reproducible-builds.org/docs/timestamps/ would say to not include dates in the build.  An alternative is to include the date of the source, e.g.<pre>git log -1 --pretty=format:%cd --date=format:+%y%m%d_%H%M_</pre>

Reply:
I do want to know when the binary actually gets built because tag/version aren't supposed 
to be reused so if the date change it's interesting to know (ie it's a one time thing)

The source tree TS is also interesting though but I expect people to copy paste the full sha if they really want to know what it is (and not helped by the version enough)